fluvio/__init__.py

```diff
@@ -170,6 +171,14 @@ def send(self, key: typing.List[int], value: typing.List[int]) -> None:
         '''
         return self._inner.send(key, value)
 
+    def send_all(self, records: typing.List[tuple[bytes, bytes]]) -> None:
```

Comment:
You might need to [`typing.Tuple`](https://docs.python.org/3/library/typing.html#typing.Tuple). The python docs don't say when `builtin.tuple` was added but my guess is that the Python 3.8 CI is failing due to this.

Reply:
I actually originally tried that but when I use `typing.tuple` the build fails for me

Reply:
Oh let me try capital-T Tuple

Reply:
Ok that worked 👍

Reply:
Yeah, the type annotations in python are a bit bleh. I only like them for the documentation generation.